src/ocean/util/log/LayoutChainsaw.d

```diff
@@ -31,6 +31,7 @@ import ocean.util.log.Log;
 
 *******************************************************************************/
 
+deprecated("This class is believe to be unused and isn't maintained")
```

Comment:
I think this is a poor deprecation message. There is no solution offered to potential users. Now that Ocean is public we can't assume nobody is using. I would just say that the class is unmaintained and as a solution I would suggest users to copy the code in their projects if they need to keep using it.